packages/app/src/app/overmind/effects/api/index.ts

```diff
@@ -62,12 +62,13 @@ export default {
   async getSandbox(id: string): Promise<Sandbox> {
     const sandbox = await api.get<Sandbox>(`/sandboxes/${id}`);
 
-    // We need to add savedCode property to track it
+    // We need to add client side properties for tracking
     return {
       ...sandbox,
       modules: sandbox.modules.map(module => ({
         ...module,
         savedCode: null,
+        isNotSynced: false,
```

Comment:
Interesting, where do we use this?

Reply:
It is being used in FilePath, ImageViewer and FuzzySearch... maybe related to iPad only?

Reply:
Oh wow, do we ever change this property? Might be a legacy thing from the redux days.

Reply:
We should track synced state on modules anyways, and make changedModuleshortids a derived, so this is a good reminder :)